tomviz/MainWindow.ui

```diff
@@ -158,7 +160,7 @@
         <bool>true</bool>
        </property>
        <property name="columnCount">
-        <number>2</number>
+        <number>1</number>
```

Comment:
This must be 2, why has this been changed to 1 again? This introduced a subtle bug before where we couldn't delete operators.